fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -630,7 +638,7 @@ public void increment(@Nonnull Count event, int amount) {
     /**
      * Get the total time spent for a given event.
      *
-     * @param event the event to get time information for
+     * @param event the event to get time information for.
```

Comment:
```suggestion
     * @param event the event to get time information for
```

Parameters (and return values) don't get periods because they are sentence fragments